docs/advanced-usage/compute-backends/kubernetes/index.md

```diff
@@ -106,5 +106,4 @@ You can set `kubernetes_memory_limit` in every `rule` under `resources`.
         kubernetes_memory_limit="8Gi"
     container:
         "docker://python:2.7-slim"
-    ...
```

Comment:
We don't need to delete this line. It means that there is more information below (`shell`).